aicord/utils/string_test.py

```diff
@@ -0,0 +1,48 @@
+import unittest
+
+from aicord.utils.string import split_paragraph_chunks
+
+
+class TestSplitParagraphChunks(unittest.TestCase):
```

Comment:
Write a test, where multiple paragraphs are smaller than max length, so that 2 paragraphs fit into 1 chunk, but 3rrd one doesn't